pkg/framework/aries/framework_test.go

```diff
@@ -390,6 +394,39 @@ func TestFramework(t *testing.T) {
 		require.Contains(t, err.Error(), "error from kms")
 	})
 
+	t.Run("test new with custom kms svc", func(t *testing.T) {
+		masterKeyFilePath := "masterKey_aries.txt"
+		tmpfile, err := ioutil.TempFile("", masterKeyFilePath)
+		require.NoError(t, err)
+
+		defer func() {
+			er := os.Remove(tmpfile.Name()) // clean up
+			require.NoError(t, er)
+		}()
```

Comment:
I really don't like it when an error variable is not `err`. Pretty of unusual. I would replace it with `defer func() { require.NoError(t, os.Remove(tmpfile.Name())) }()`

Reply:
removed variable